src/main.rs

```diff
@@ -112,19 +119,27 @@ pub async fn main() -> Result<(), anyhow::Error> {
         Some(m) => std::path::PathBuf::from(m),
         None => tempfile::tempdir()?.into_path(),
     };
+
+    let log_dir = match matches.value_of("log_dir") {
+        Some(m) => std::path::PathBuf::from(m),
+        None => tempfile::tempdir()?.into_path(),
```

Comment:
Is the directory deleted after Wagi stops?
Would it make sense to log the path to the console?

Reply:
It is not deleted because we consume the `TempDir` with `into_path`, but logging the path is probably a good idea